src/main/java/com/softserve/academy/spaced/repetition/service/DeckService.java

```diff
@@ -76,18 +76,18 @@ public void updateDeck(Deck updatedDeck, Long deckId, Long categoryId) {
 
     @Transactional
     public void deleteDeck(Long deckId) {
-        List<Course> courses = courseRepository.findAll();
-        List<Deck> decks;
-        for(Course course: courses){
-            decks=course.getDecks();
-            for(Deck deck : decks){
-                if (deck.getId().equals(deckId)) {
-                    decks.remove(deck);
-                    break;
-                }
-            }
-            courseRepository.save(course);
-        }
+//        List<Course> courses = courseRepository.findAll();
+//        List<Deck> decks;
+//        for(Course course: courses){
+//            decks=course.getDecks();
+//            for(Deck deck : decks){
+//                if (deck.getId().equals(deckId)) {
+//                    decks.remove(deck);
+//                    break;
+//                }
+//            }
+//            courseRepository.save(course);
+//        }
```

Comment:
I'm sure, we don't need all of these comments here.

Reply:
Yesterday you said it is wrong to delete card this way.

Reply:
Well, then it's better to either remove these comments or make them work correctly or leave them for now. But it's my opinion, I'm not sure. Answer to the previous comments better, please.